src/legacy/core_plugins/kibana/public/dashboard/dashboard_app_controller.tsx

```diff
@@ -128,6 +128,8 @@ export class DashboardAppController {
     const panelActionsRegistry = Private(ContextMenuActionsRegistryProvider);
     const getUnhashableStates = Private(getUnhashableStatesProvider);
     const shareContextMenuExtensions = Private(ShareContextMenuExtensionsRegistryProvider);
+    const abortController = new AbortController();
```

Comment:
I think you're going to want to import `abortcontroller-polyfill` here as some browsers don't support this natively.

Reply:
Also I think you're going to want to create a new `AbortController` for each update, not just one on the controller load.

Reply:
@lukasolson I had thought about the polyfill question too, because I was worried that any plugin dev who wanted to use this feature would need to deal with the polyfill. But then I noticed we are importing it here: https://github.com/elastic/kibana/blob/master/src/legacy/ui/ui_bundles/app_entry_template.js#L40

I'm not super familiar with how the entry template works, but I assume this means we won't need to re-import that polyfill elsewhere in kibana?

Reply:
Yeah, you could totally be right. I'll give this PR a spin on IE just to make sure.